api/src/main/java/eu/coatrack/api/EntryPoint.java

```diff
@@ -39,7 +39,7 @@
     private double pricePerCall;
 
     @OrderColumn
-    private Integer position;
+    private int position;
```

Comment:
In the production system there is probably old/legacy data where `position` is `null`. This change will require to add values to the legacy data before as a pre-condition to deploy/run the new version. 

As we do not now the expected/preferred ordering for legacy data, another solution would be more elegant, where old records can keep a `null` position value until the users (re-)order them. If such an approach is not possible, we will need a database migration script in order to set position values in legacy data.

Reply:
The first solution will not work, because when the position is null and a JSON is parsed, it will always retrieve a java null exception, however, your second solution seems good to me.
One question, when you say a database migration script, do you mean in SQL?